src/test/scala-2.11/markets/clearing/engines/CDAMatchingEngineSpec.scala

```diff
@@ -316,14 +315,14 @@ class CDAMatchingEngineSpec extends TestKit(ActorSystem("CDAMatchingEngineSpec")
       val askPrice = randomLong(prng, upper = bidPrice)
       val askQuantity = randomLong(prng, lower = bidQuantity)
       val askOrder = LimitAskOrder(askOrderIssuer, askPrice, askQuantity, randomLong(prng), testTradable, uuid)
-      val fills = matchingEngine.findMatch(askOrder)
+      val matchings = matchingEngine.findMatch(askOrder)
 
-      Then("the matching engine should generate a PartialMatch")
+      Then("the matching engine should generate a Matching")
 
       val residualAskQuantity = askQuantity - bidQuantity
-      val (filledAskOrder, residualAskOrder) = askOrder.split(residualAskQuantity)
-      val fill = PartialMatch(bidOrder, filledAskOrder, bidPrice)
-      fills should equal(Some(immutable.Queue[Match](fill)))
+      val (_, residualAskOrder) = askOrder.split(residualAskQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1545342729/issues/source?bid=2921361&fileBranchId=3015559#l323)